src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/SummaryModel.java

```diff
@@ -57,6 +57,9 @@
     private Integer clusters;
     private Long sockets;
     private Integer vms;
+    private Integer hosts;
```

Comment:
Could you add the new fields to the `toString` method as well and remove the `'\'' +` please?

Reply:
@mrizzi done